evac/worker.py

```diff
@@ -706,7 +709,6 @@ def move_agents(self, agents_to_move, step, time):
         for floor_num in range(len(self.floors)):
             agents_who_leave_current_floor_indexes = [agent[3] for agent in agents_to_move if agent[0] == floor_num]
             if agents_who_leave_current_floor_indexes:
-                self.floors[floor_num].rset = time
```

Comment:
Potential issue with RSET will be explored in issue #369. Please, restore those changes.